src/html/mutation_details_table_view.html

```diff
@@ -43,69 +43,65 @@
 </script>
 
 <script type="text/template" id="mutation_table_mutation_type_template">
-	<span class='{{mutationTypeClass}}'>
-		<label>{{mutationTypeText}}</label>
-	</span>
+	<span class='{{mutationTypeClass}}'>{{mutationTypeText}}</span>
 </script>
 
 <script type="text/template" id="mutation_table_cna_template">
-	<label alt='{{cnaTip}}' class='simple-tip {{cnaClass}}'>{{cna}}</label>
+	<span alt='{{cnaTip}}' class='simple-tip {{cnaClass}}'>{{cna}}</span>
 </script>
 
 <script type="text/template" id="mutation_table_cosmic_template">
-	<label class='{{cosmicClass}}'>{{cosmicCount}}</label>
+	<span class='{{cosmicClass}}'>{{cosmicCount}}</span>
 </script>
 
 <script type="text/template" id="mutation_table_mutation_status_template">
 	<span alt='{{mutationStatusTip}}' class='simple-tip {{mutationStatusClass}}'>
-		<label>{{mutationStatusText}}</label>
+		{{mutationStatusText}}
 	</span>
 </script>
 
 <script type="text/template" id="mutation_table_validation_status_template">
 	<span alt='{{validationStatusTip}}' class="simple-tip {{validationStatusClass}}">
-		<label>{{validationStatusText}}</label>
+		{{validationStatusText}}
 	</span>
 </script>
 
 <script type="text/template" id="mutation_table_mutation_assessor_template">
-	<span class='{{omaClass}} {{fisClass}}'>
-		<label>{{fisText}}</label>
-	</span>
+	<span class='{{omaClass}} {{fisClass}}'>{{fisText}}</span>
 </script>
 
 <script type="text/template" id="mutation_table_start_pos_template">
-	<label class='{{startPosClass}}'>{{startPos}}</label>
+	<span class='{{startPosClass}}'>{{startPos}}</span>
 </script>
 
 <script type="text/template" id="mutation_table_end_pos_template">
-	<label class='{{endPosClass}}'>{{endPos}}</label>
+	<span class='{{endPosClass}}'>{{endPos}}</span>
 </script>
 
 <script type="text/template" id="mutation_table_tumor_freq_template">
-	<label alt='<b>{{tumorAltCount}}</b> variant reads out of <b>{{tumorTotalCount}}</b> total'
-	       class='{{tumorFreqClass}} {{tumorFreqTipClass}}'>{{tumorFreq}}</label>
+	<span title='{{tumorAltCount}} variant reads out of {{tumorTotalCount}} total'
```

Comment:
This breaks the qTip initialization. It is configured to read the content from the `alt` tag. The tooltip is still displayed though, but now in a default tooltip box instead of a qTip box.